ts/sagas/wallet/cobadgeReminder.ts

```diff
@@ -0,0 +1,96 @@
+import { put, select, take } from "redux-saga/effects";
+import * as pot from "italia-ts-commons/lib/pot";
+import { getType, isActionOf } from "typesafe-actions";
+import {
+  bancomatListVisibleInWalletSelector,
+  cobadgeListVisibleInWalletSelector
+} from "../../store/reducers/wallet/wallets";
+import {
+  BancomatPaymentMethod,
+  CreditCardPaymentMethod
+} from "../../types/pagopa";
+import { coBadgeAbiConfigurationSelector } from "../../features/wallet/onboarding/cobadge/store/reducers/abiConfiguration";
+import { IndexedById } from "../../store/helpers/indexer";
+import { StatusEnum } from "../../../definitions/pagopa/cobadge/configuration/CoBadgeService";
+import { NetworkError } from "../../utils/errors";
+import { loadCoBadgeAbiConfiguration } from "../../features/wallet/onboarding/cobadge/store/actions";
+import { sendAddCobadgeMessage } from "../../store/actions/wallet/wallets";
+
+/**
+ * This saga aims to send an event to Mixpanel to get information about whether the user has a bancomat card with which
+ * it is allowed to add a co-badge card and has not yet done or not.
+ *
+ * This saga is called only if the {@link bancomatListVisibleInWalletSelector} return some
+ */
+export function* sendAddCobadgeMessageSaga() {
+  // Check if there is at least one bancomat
+  const maybeBancomatListVisibleInWallet: pot.Pot<
+    ReadonlyArray<BancomatPaymentMethod>,
+    Error
+  > = yield select(bancomatListVisibleInWalletSelector);
+
+  const bancomatListVisibleInWallet = pot.getOrElse(
+    maybeBancomatListVisibleInWallet,
+    []
+  );
+
+  if (bancomatListVisibleInWallet.length === 0) {
+    yield put(sendAddCobadgeMessage(false));
+    return;
+  }
+
+  // Extract the cobadgeAbi if there is at least a cobadge card
+  const maybeCobadgeVisibleInWallet: pot.Pot<
+    ReadonlyArray<CreditCardPaymentMethod>,
+    Error
+  > = yield select(cobadgeListVisibleInWalletSelector);
+
+  const cobadgeVisibleInWallet = pot.getOrElse(maybeCobadgeVisibleInWallet, []);
+  const cobadgeAbis = cobadgeVisibleInWallet
+    .filter(c => c.info.issuerAbiCode !== undefined)
```

Comment:
Is this filter needed?
it is already done by `cobadgeListVisibleInWalletSelector`. Isn't it?

Reply:
`maybeCobadgeVisibleInWallet`
`cobadgeVisibleInWallet`
`cobadgeAbis`
can be computed just before `enalbedAbis`
If the code returns before that point you have executed few instructions uselessly

Reply:
For that concern `cobadgeListVisibleInWalletSelector`, yes it's true.

Both modified in [de146d1](https://github.com/pagopa/io-app/pull/2952/commits/de146d1f1d46b18931915c27f001ad8e3b22d722)